source/computer/macos-setup.rst

```diff
@@ -1,9 +1,251 @@
+类似于 Windows 下的 `Microsoft Visual Studio <https://visualstudio.microsoft.com/>`__\ 。
+Command Line Tools for Xcode 是 Xcode 的一部分，其包含了常用的命令行开发工具，
+比如 C 编译器、\ ``make``\ 、\ ``git`` 等，是 macOS 下编程开发的必须软件。
+
+按下 :kbd:`Command+空格`\ ，搜索“Terminal”并按下 :kbd:`Enter` 键以打开 Terminal 应用。
+在 Terminal 中执行如下命令以安装 Command Line Tools for Xcode::
+
+   $ xcode-select --install
+
+在弹出的窗口中点击“Install”即可。
+
+此处安装的 Command Line Tools for Xcode 可能不是最新版。点击左上角的 Apple 图标，
+在“系统偏好设置”的“软件更新”中查看是否有相关更新。如果有，则升级到最新版。
+
+.. note::
+
+   Command Line Tools for Xcode 会被安装到 :file:`/Library/Developer/CommandLineTools/`
+   目录，其提供的命令行工具位于 :file:`/Library/Developer/CommandLineTools/usr/bin` 目录，
+   包括 ``gcc``、``make``、``git`` 等。
+
+Homebrew
+^^^^^^^^
+
+`Homebrew <https://brew.sh/index_zh-cn.html>`__ 是 macOS 下最流行的第三方软件包管理器，
+类似于 Linux 系统下的 ``apt``、``yum``、``dnf`` 等包管理器。
+其可以用于安装各种常见的软件包、库文件以及字体。日常及科研工作中所需的大多数软件、
+库文件以及字体都可以通过 Homebrew 安装。
+
+安装
+""""
+
+要安装 Homebrew，需要将如下命令复制到终端中并执行::
+
+    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+Homebrew 会被安装到 :file:`/usr/local/` 目录下。通过 Homebrew 安装的所有软件包
+也都将被安装到该目录下。
+
+.. note::
+
+   GitHub 在国内访问不畅，以上安装命令可能会由于网络问题而失败。
+   若以上命令失败，可参考 https://github.com/ineo6/homebrew-install 的
+   解决办法。
+
+   同样由于网络的原因，Homebrew 在国内可能下载速度较慢。建议参照
+   https://github.com/ineo6/homebrew-install 教程设置中科大源或清华大学源。
+
+使用
+""""
+
+安装好 Homebrew 后，打开新的 Terminal 便可以使用 Homebrew 提供的 ``brew`` 命令。
+``brew`` 的详细用法见\ `官方文档 <https://docs.brew.sh/Manpage>`__\ 。此处仅列出一些常用的用法::
+
+    # 模糊搜索与 wget 相关的软件
+    $ brew search wget
+
+    # 安装 wget 软件包
+    $ brew install wget
+
+    # 安装 Visual Studio Code
+    # 由于 Visual Studio Code 是带图形界面的软件，因而这里需要使用 --cask 选项
+    $ brew install --cask visual-studio-code
+
+    # 升级某个软件
+    $ brew upgrade xxx
+
+    # 卸载某个软件
+    $ brew uninstall xxx
+
+.. note::
+
+   ``brew``\ ：Homebrew 用于安装各种软件包、库文件以及字体等的命令。
+
+   ``Formula``\ ：一个软件的描述文件，包含了如何安装此软件等信息。``Formulae`` 是其复数。
+   每个软件对应一个 ``Formula``\ 。例如，git 对应 ``Formula`` 是
+   :file:`/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/git.rb`\ 。
+   使用 ``brew ls --formula`` 命令可以查看已安装的 ``Formulae``。
+
+   ``Tap``\ ：一个含有一系列软件（即 ``Formulae``\ ）的 git 仓库。使用
+   `brew tap <https://docs.brew.sh/Taps#the-brew-tap-command>`__
+   命令查看已启用的仓库列表或启用仓库。已启用的仓库位于
+   :file:`/usr/local/Homebrew/Library/Taps/homebrew/` 目录。
+   常见软件仓库有 ``homebrew-core`` 和 ``homebrew-cask``\ 。其中，\ ``homebrew-core`` 是内置
+   核心仓库，\ ``homebrew-cask`` 仓库则含有各种 macOS 系统下的应用程序。
+
+   ``Bottle``\ ：预先编译好的二进制软件包。使用 ``Bottle`` 安装软件，
+   比从源码编译和安装更快。如果一个软件仓库包含预编译的软件包，使用 ``brew install``
+   时会自动使用它。
+
+   ``Cask``\ ：Homebrew 的扩展（extension ），用来安装 macOS 的图形界面应用程序。
+   使用 ``brew ls --cask`` 命令可以查看已安装的 ``casks``。
+
+   ``Cellar``\ ：所有软件的安装目录，即 :file:`/usr/local/Cellar`\ 。
+
+   ``Keg``\ ：某一软件的安装目录，如 :file:`/usr/local/Cellar/git/2.30.0`\ 。
+
+   更详细的解释请查看\ `官方文档 <https://docs.brew.sh/Formula-Cookbook#homebrew-terminology>`__\ 。
```

Comment:
Two comments:

1. Formula el al., are not commands. It's inappropriate to use double-backticks.
2. It's better to use the definition list (see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks)

Reply:
Fixed